### PR TITLE
thefuck 3.31

### PR DESCRIPTION
**Pull request by scorphus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

New releaase fixes a [vulnerability][1].

[1]: https://github.com/nvbn/thefuck/pull/1206

### CODE REVIEW

File: Formula/thefuck.rb [hunk 1]
```diff
@@ -3,20 +3,12 @@ class Thefuck < Formula
...
   head "https://github.com/nvbn/thefuck.git"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_big_sur: "3bc7586bc2ff253689bfe24ff35af100c6659605dddaefbec92cec29e2a25389"
-    sha256 cellar: :any_skip_relocation, big_sur:       "00fb2bb9fbd724cf47c75e99ffc44a356587cd34d423feefe21ec1b57f954133"
-    sha256 cellar: :any_skip_relocation, catalina:      "7f8ecd031ae58cc414d695058d7332b4b93c8e2959cf5114c82609d5676b24a7"
-    sha256 cellar: :any_skip_relocation, mojave:        "b7ff0b5553c12c27d3311d557604b708e0e4ea0c315b03198f354667b057a1c8"
-    sha256 cellar: :any_skip_relocation, high_sierra:   "e2d94b4c4d959f39370227a2a029956f85b5ef6e739bb0b55969ca2b0c034bb5"
-  end
-
```
Comment (nandahkrishna) [inline Formula/thefuck.rb:19]: Please don't touch the bottle block, as mentioned in the documentation and contributing guidelines.

Comment (nandahkrishna) [reply Formula/thefuck.rb]: Perhaps you used `--new` – that is only for new formulae.

Comment (scorphus) [reply Formula/thefuck.rb]: Yup. That was it. Won't happen again.

**Comment by nandahkrishna** [comment]
@scorphus it seems like a few resources need updates here. Could you use `brew update-python-resources thefuck` to do this, and squash the changes with your existing commit?

Note: Using `brew bump-formula-pr` will do this automatically for you, and all you have to do (usually) is specify `--version <VERSION>` and the formula name.

File: Formula/thefuck.rb (hunk 1 above)
Comment (scorphus) [reply Formula/thefuck.rb]: BTW, should (or even must) I update those dependencies?

Comment (nandahkrishna) [reply Formula/thefuck.rb]: No issues!

> BTW, should (or even must) I update those dependencies?

Yes, please see my comment below (and also remove `revision 1`).

**Comment by scorphus** [comment]
`brew bump-formula-pr --strict thefuck --version=3.31` didn't work for me:

```
Error: You need to bump this formula manually since the new URL
and old URL are both:
  https://files.pythonhosted.org/packages/21/d2/f5194c2d51b2aab4bda6de97c5eb1ca5a547ad9b9f778ab5bf4ce19fba88/thefuck-3.31.tar.g
```

What am I doing wrong?

**Comment by nandahkrishna** [comment]
I should've also mentioned that the command is used to create new PRs, and not for this one. That was meant to be for your next contribution 😅

**Comment by scorphus** [comment]
Yeah, I meant that I tried with `bump-formula-pr` before doing it manually.

**Comment by nandahkrishna** [comment]
It seems like you updated the URL before you ran `bump-formula-pr`, hence the error. The command should be run without making any modifications to the formula.

**Comment by scorphus** [comment]
That wasn't the case. But it should be reproducible until this PR is not accepted, right?

Anyways, hopefully that last push has it all. And I'll try `bump-formula-pr` again in the next release — y then I hope I'll be a bit luckier.

**Review by nandahkrishna — APPROVED** [review]
Thanks!

**Comment by scorphus** [comment]
Thanks for the help, @nandahkrishna 👍

**Pull request closed without merging**